docs/source/status.rst

```diff
@@ -126,6 +126,8 @@ IPC Format
 +-----------------------------+-------+-------+-------+------------+-------+-------+-------+
 | Sparse tensors              | ✓     |       |       |            |       |       |       |
 +-----------------------------+-------+-------+-------+------------+-------+-------+-------+
+| Buffer compression          | ✓     |       |       |            |       |       | ✓     |
++-----------------------------+-------+-------+-------+------------+-------+-------+-------+
```

Comment:
@quinnj AFAICT, Julia supports IPC buffer compression?
@nevi-me AFAICT, Rust doens't support IPC buffer compression?

Reply:
No Rust support yet. Came across issues while working on it, but now that there are test files, I'll be able to finish the support

Reply:
Yes, Julia supports IPC buffer compression/decompression for lz4_frame and zstd.